### PR TITLE
Enable NTO 4.12+ nightly testing

### PR DESCRIPTION
**Pull request by jmencak**
Other changes:
  - remove 4.8 periodic testing
  - lower the frequency of 4.10 testing
  - add forgotten `make vet` to 4.12+

### CODE REVIEW

**Comment by jmencak** [comment]
Only if you have a minute to go over whether the changes make sense to you, @kpouget  ?

File: ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-release-4.10.yaml [hunk 1]
```diff
@@ -76,7 +76,7 @@ tests:
     cluster_profile: aws
     workflow: openshift-upgrade-aws
 - as: e2e-nightly
-  cron: 0 23 * * *
+  cron: 47 21 14,25 * *
```
Comment (fromanirh) [inline ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-release-4.10.yaml:79]: this seems very very specific, any particular reason to run it at this time and way less often than before?

**Review by fromanirh — COMMENTED** [review]
besides the very specific timing of the cronjob, which caught my eye, everything else seems straightforward

File: ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-release-4.10.yaml (hunk 1 above)
Comment (jmencak) [reply ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-release-4.10.yaml:79]: Basically, to save money.  There was a PR in the past to reduce the frequency.  That specific time/date was picked from that PR.

Comment (jmencak) [reply ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-release-4.10.yaml:79]: See: https://github.com/openshift/release/pull/31193

Comment (fromanirh) [reply ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-release-4.10.yaml:79]: ok then!

**Comment by fromanirh** [comment]
LGTM

**Comment by jmencak** [comment]
@smg247 I've seen this: https://github.com/openshift/release/pull/34417

How do I get the rehearsals-ack label?

**Comment by smg247** [comment]
It looks like the hook got dropped and the plugin missed your PR for the usage message.
You can do it with the `/pj-rehearse ack` command.
/pj-rehearse ack

**Comment by kpouget** [comment]
> It looks like the hook got dropped and the plugin missed your PR for the usage message.

I was going to share this link which gave me some hint (for rehearsal is disabled for my repo, so that's a bit different)
https://github.com/openshift/release/pull/34152#issuecomment-1320960423

**Pull request merged**